js/libs/ui-shared/src/controls/FormLabel.tsx

```diff
@@ -26,6 +26,10 @@ export const FormLabel = ({
   ...rest
 }: PropsWithChildren<FormLabelProps>) => (
   <FormGroup
+    onFocus={() => {
+      const input = document.activeElement as HTMLInputElement;
+      if (input.value) input.select();
+    }}
```

Comment:
Why would we want to auto-focus here? If this is for accessibility reasons this should be handled upstream in PatternFly, not here in this project. Even then, I don't really see a valid reason for doing this, can you elaborate?

Reply:
My intention was to mainly apply it to the select control and not generically. With the non-typeahead control (which you will get with less than 10 entries) you can simply click on the control and it shows you the list of values. However, for the typeahead control, if there is an existing value then clicking on the control puts the cursor at the end of the existing value. If you want to change the value you then have to manually delete the current value before you can start typing what you want to search for. By selecting the current value the user can just start typing.

I agree that it should ideally be a patternfly thing, but I was hoping to minimize the blast-radius of the change